pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -88,20 +110,21 @@ public InputStream getStream() throws IOException {
     if (stream != null) {
       return stream;
     }
-    StreamWrapperFinalizeAction finalizeAction = this.finalizeAction;
+    LazyCleaner.CleaningAction finalizeAction = this.finalizeAction;
     if (finalizeAction != null) {
-      return finalizeAction.getStream();
+      InputStream stream = ((StreamWrapperCleaningAction) (finalizeAction)).getStream();
```

Comment:
Can we avoid class cast here? I am afraid it would fail in runtime.

Reply:
I wasn't able to create a StreamWrapperCleaningAction lambda. I'm sure there is a way but I couldn't figure it out